### ISSUE TITLE
Support for DynamoDB

### ISSUE THREAD

**Issue by stevenpila** [jobrunr/jobrunr#108]
Hi,

Is there any support or plan for AWS DynamoDB?

Thanks.

**Comment by rdehuyss** [comment]
Hmmm, I wasn't really planning to, to be honest. I try to support the top 10 databases (according to https://db-engines.com/en/ranking) - except for MS Access off-course :smile: . 

If you want, I can add support for it but this will be in time & material mode. Supporting this open-source project and all the different DB's takes a lot of time (next to my day to day job). You're off-course welcome to create a PR.

**Comment by stevenpila** [comment]
Oh I see, that's okay but would really love to have that feature in the future. :)

**Comment by aguerard** [comment]
@rdehuyss do you estimate it would take a lot of work to implement a DDB “driver”? I’d be interested in potentially taking this up in my free time if it’s on the order of a few weeks (vs months)

**Comment by rdehuyss** [comment]
Well, it depends on the capabilities of DynamoDB. I've implemented MongoDB
and that was finished in a couple of days whereas I tried implementing
Cassandra which failed and felt almost impossible.

I would recommend having a look at the different StorageProviders that
already exist - if you know DynamoDB, your estimate will probably be better
than mine.

Support for adding most databases was not that hard (more in the range of
days / perhaps max a week) including all the tests, ... .

Good luck! And do reach out if you have any questions.
